.github/dependabot.yml

```diff
@@ -7,6 +7,10 @@ updates:
       time: "09:00"
       timezone: "UTC"
     open-pull-requests-limit: 10
+    groups:
+      all-dependencies:
+        patterns:
+          - "*"
```

Comment:
does that group multiple dep updates in a single PR?

I don't mind a lot of PRs, and I think keeping the granularity is better